neo_force_scheme/neo_force_scheme.py

```diff
@@ -164,10 +164,20 @@ def _fit(self, X,
 
 
     def _transform(self, X, *, index, total, inplace, n_dimension: Optional[int] = 2, force_projection_dimensions=None,
-                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0)):
+                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0),
+                   confidence_interval: Optional[float] = 1):
         # iterate until max_it or if the error does not change more than the tolerance
         error = math.inf
 
+        z_score = 1
```

Comment:
to what end do we use the z_score?

Reply:
z_score is used in the iteration() and is to determine the shape of the gaussian function (), and the details of the logic is mentioned is the pictures above

Reply:
got it! I think it is better to change its name then.. z-score in machine learning is something similar to the kruskel score (another way of giving a final score to a model or a visualization). This will confuse a bunch of people. Can you change the variable name?

Reply:
variable name is changed to gaussian_function_z_score